src/workflows/loop-timer.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  ParentClosePolicy,
+  sleep,
+  startChild,
+  continueAsNew,
+} from '@temporalio/workflow';
+import { importDocmaps } from './import-docmaps';
+
+export async function loopTimer(docMapIndexUrl: string, sleepTime: string = '1 hour', hashes: string[] = []): Promise<void> {
```

Comment:
I think this could do with a better name (similar to my comments on the README). `pollDocmapIndex`?